Replicator/c4ReplicatorImpl.hh

```diff
@@ -296,8 +296,8 @@ namespace litecore {
             }
 
             setStatusFlag(kC4Suspended, false);
-            logInfo("Starting Replicator %s with config: {%s}\n", _replicator->loggingName().c_str(),
-                    std::string(*_options).c_str());
+            logInfo("Starting Replicator %s with config: {%s} with endpoing: %.*s", _replicator->loggingName().c_str(),
+                    std::string(*_options).c_str(), SPLAT(_replicator->remoteURL()));
```

Comment:
What will happen if the replicator is for db-to-db or p2p sync?

Reply:
Whenever Replicator is instantiated it sets `_remoteURL = webSocket->url()`. So I believe this would be something like `ws://loopback`. I will run test and verify.

Reply:
DB to DB endpoint looks like `file:////var/folders/dq/4702h1qn7wdclczftpkmv1dr0000gp/T/LiteCore_Tests_1729177323.cblite2/cbl_core_test2.cblite2.cblite2/ `

Reply:
Oh yeah this is ReplicatorImpl so for P2P the implementation is by platform - so this exact log isn't in the P2P replicators.

Reply:
Not sure what we should do about this - @pasin any ideas? Should LiteCore be logging the Replicator options from P2P replicator like it does for remote replicator?

Reply:
Not sure I understand this. The c4ReplicatorImpl's start() should be used by any types of the replicator. I think we can focus on what is being logged here with the remoteURL() with the replicator with message endpoint is used. Maybe the easiest way to find out is to look at the log when running the message endpoint replicator tests from the platform (e.g. iOS) that uses LiteCore in this PR?

Reply:
I've also seen this for the passive replicator 
```endpoint: x-msg-conn:///0x104397e20```

Reply:
This is good enough.